### PR TITLE
Feat/dashboard

### PR DESCRIPTION
**Pull request by ex3emxl**

### CODE REVIEW

File: configs/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            version: package.version
+        }),
+        new MiniCssExtractPlugin({filename:'style.css'}),
+        new webpack.ProvidePlugin({
+            React: 'react',
+            Component: ['react', 'Component']
+        })
+];
+
+if(isFileCss){
```
Comment (iliatcymbal) [inline configs/webpack.config.js]: please, use spaces to separate operators, brackets etc.

File: configs/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    plugins.push(new MiniCssExtractPlugin({filename:'style.css'}));
```
Comment (iliatcymbal) [inline configs/webpack.config.js]: space before after and before curly brackets

File: src/app.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import ReactDom from 'react-dom';
+
+import Main from './components/main';
+
+import './components/general.scss';
+
+const Wrapper = (
+    <>
+        <Main title="Title" />
+    </>
```
Comment (iliatcymbal) [inline src/app.js]: you don't need here react fragment `<>...</>` (because you have only one component)

File: src/app.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+);
+
+
+
```
Comment (iliatcymbal) [inline src/app.js]: too many empty lines

File: src/components/dashboard/Dashboard.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+class Dashboard extends Component {
+
+    state = {
+        done: 0,
+        inProgress: 0,
+        waiting: 0
+    };
+
+    render() {
+        const {done, inProgress, waiting} = this.state;
```
Comment (iliatcymbal) [inline src/components/dashboard/Dashboard.js]: spaces after and before `{}`

File: src/components/main/Main.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Content from '../content';
+
+import './main.scss';
+
+
+const Main = () => (
+    <main className='main'>
+        <Content/>
```
Comment (iliatcymbal) [inline src/components/main/Main.js]: space before `/>`

File: src/components/hello/Hello.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```
Comment (iliatcymbal) [inline src/components/hello/Hello.js:1]: why you need `Hello.js`. Where is it used? If you don't need it, remove

**Review by iliatcymbal — CHANGES_REQUESTED** [review]
Please, see my notes. Don't forget about code formatting

File: src/components/hello/Hello.js (hunk 7 above)
Comment (ex3emxl) [reply src/components/hello/Hello.js:1]: This used in Content.js accorded to require

**Review by iliatcymbal — APPROVED** [review]
looks good to me

**Pull request merged**